src/js/app.js

```diff
@@ -50,8 +51,13 @@ const app = new Vue({
 
       <radical-selection v-bind:radical_selection="radical_selection"
                          v-on:select-radical="selectRadical"></radical-selection>
+    </section>
 
-      <about-section></about-section>
+    <about-section></about-section>
+
+    <footer class="footer">
+      <p>Copyright © 2019 - <a class="about-link" href="https://github.com/rewhowe/kanji">Kanji</a></p>
```

Comment:
any name other than "Kanji"

Reply:
Oh. Actually, I thought GitHub would be fine since it's a link to the GitHub repo.

(Though that is a good point, there isn't really any fancy name for this tool... w)